packages/rocketchat-crowd/server/crowd.js

```diff
@@ -75,15 +76,68 @@ export class CROWD {
 			return;
 		}
 
-		logger.info('Going to crowd:', username);
-		const auth = this.crowdClient.user.authenticateSync(username, password);
+		logger.info('Extracting crowd_username');
+		let user = null;
+		let crowd_username = username;
+		try {
+			user = Meteor.users.findOne({
+				username,
+			});
+			crowd_username = user.crowd_username;
+		} catch (error) {
+			logger.debug(error);
```

Comment:
Please use if instead of try catch since the error trying to access `crowd_username` of undefined isn't useful

Reply:
Done